### PR TITLE
Switch from grunt to babel & add tradutore support

### LINKED ISSUE #3: Remove Grunt tasks
* [x] Use Traduttore for translations
* [x] Use plain Uglify instead of Grunt task

### PR DESCRIPTION
**Pull request by swissspidy**
* No Grunt
* Traduttore

Fixes #3.

### CODE REVIEW

File: js-comment-form-validation.php [hunk 1]
```diff
@@ -32,29 +31,40 @@
...
+/**
+ * Sets up translation downloads for the plugin.
+ *
+ * @since 1.2.0
+ */
+function init_traduttore() {
+	add_project(
+		'plugin',
+		'js-comment-form-validation',
+		'https://translate.example.com/api/translations/required/js-comment-form-validation/'
```
Comment (grappler) [inline js-comment-form-validation.php]:
```suggestion
		'https://translate.required.com/api/translations/required/js-comment-form-validation/'
```

File: js-comment-form-validation.php [hunk 2]
```diff
@@ -32,29 +31,40 @@
...
+add_action( 'init', __NAMESPACE__ . '\init_traduttore' );
 
 /**
  * Registers and enqueues the comment validation scripts.
  *
  * @since 1.0.0
  */
 function enqueue_scripts() {
-	wp_register_script( 'jquery-validation', plugin_dir_url( __FILE__ ) . 'js/jquery.validate.min.js', [ 'jquery' ], '1.16.0', true );
+	wp_register_script( 'jquery-validation', plugin_dir_url( __FILE__ ) . 'assets/js/vendor/jquery.validate.min.js', [ 'jquery' ], '1.16.0', true );
```
Comment (grappler) [inline js-comment-form-validation.php:63]: This file is missing.

File: package.json [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/preset-env": "^7.1.0",
+    "uglify-js": "^3.4.9"
   },
-  "keywords":        []
+  "scripts": {
+    "uglifyjs": "find assets/js -type f -name '*.js' ! -name '*.min.js' -maxdepth 1 | while read FILE ; do uglifyjs \"${FILE}\" -o \"${FILE%.js}.min.js\" ; done",
+    "build": "babel assets/js/src -d assets/js && npm run uglifyjs",
+    "watch": "nodemon --watch assets/js/src -e js -x 'npm run build'"
```
Comment (grappler) [inline package.json:25]: It would be good to document these commands in the readme.

**Review by grappler — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (swissspidy) [reply package.json:25]: IDK. These commands are specifically for us developers.

The readme is for users installing this plugin.

If anything, this should go to a CONTRIBUTING.md file or pull request template or similar.

Comment (grappler) [reply package.json:25]: `CONTRIBUTING.md` would not be too bad.

File: js-comment-form-validation.php [hunk 4]
```diff
@@ -48,7 +48,7 @@ function init_traduttore() {
 	add_project(
 		'plugin',
 		'js-comment-form-validation',
-		'https://translate.example.com/api/translations/required/js-comment-form-validation/'
+span class="pl-s1"> 		'https://translate.required.com/api/translations/required/js-comment-form-validation/'
```
Comment (grappler) [inline js-comment-form-validation.php]: This did not seem to work. 😞

File: package.json (hunk 3 above)
Comment (swissspidy) [reply package.json:25]: Alright. Can be done in a separate PR IMO.

**Review by grappler — APPROVED** [review]

**Pull request merged**